astropy/coordinates/sky_coordinate_parsers.py

```diff
@@ -659,41 +653,29 @@ def _parse_ra_dec(coord_str):
     ----------
     coord_str : str
         Coordinate string to parse.
+    is_ra_dec_representation : bool
```

Comment:
Could we add to the docstring which formats are supported for RA/Dec only?

Reply:
I've updated the docstring.